### PR TITLE
Problem:(CRO-483) There is no dedicated ci builder

### PR DESCRIPTION
**Pull request by leejw51**
Solution: add drone.io  config
add prepare.sh

### CODE REVIEW

**Comment by leejw51** [comment]
it took about 15 minutes on i7

**Comment by leejw51** [comment]
```
```

**Comment by leejw51** [comment]
if we use local docker image using "pull:never"
compile + unit-test total time is
3m 40s
'''
steps:
- name: test
  image: chain
  pull: never
'''

File: install_zeromq.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+FILE=/home/travis/lib/libzmq.so
```
Comment (tomtau) [inline install_zeromq.sh]: does this work on drone?

Comment (leejw51) [reply install_zeromq.sh]: yes, this script will be used in drone,too.

Comment (tomtau) [reply install_zeromq.sh]: but this assumes there's a "travis" user?

Comment (leejw51) [reply install_zeromq.sh]: yes, also drone.io uses it

**Comment by leejw51** [comment]
drone io is workig now.
will move to azure soon.

File: .drone.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+kind: pipeline
+name: default
+
+steps:
+- name: test
+  image: chain
```
Comment (tomtau) [inline .drone.yml]: is there a docker file for this?

File: .drone.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  commands:
+  - export RUST_BACKTRACE=1
+  - export RUSTFLAGS=-Ctarget-feature=+aes,+ssse3
+  - export LD_LIBRARY_PATH=$HOME/lib
+  - export LIBRARY_PATH=$HOME/lib
+  - export PATH=$HOME/.cargo/bin:$HOME/.local/bin:$PATH
+  - export PKG_CONFIG_PATH=$HOME/lib/pkgconfig
+  - cargo build
+  - cargo test
+  - echo "HOME FOLDER= " $HOME
```
Comment (tomtau) [inline .drone.yml]: this doesn't call those preparation bash scripts -- are they needed?

File: .drone.yml (hunk 2 above)
Comment (leejw51crypto) [reply .drone.yml]: yes , i already setup inside azure

**Pull request merged**